vagrant/scripts/install_iml_local.sh

```diff
@@ -4,8 +4,12 @@ set -ex
 
 yum copr enable -y managerforlustre/manager-for-lustre-devel
 # Get latest rpmdevtools
-yum install -y https://copr-be.cloud.fedoraproject.org/results/managerforlustre/buildtools/epel-8-x86_64/01152137-rpmdevtools/rpmdevtools-8.10-7.el8.noarch.rpm
-yum install -y https://download.postgresql.org/pub/repos/yum/reporpms/EL-7-x86_64/pgdg-redhat-repo-latest.noarch.rpm
+if ! yum list installed rpmdevtools-8.10-7.el8.noarch; then
+    yum install -y https://copr-be.cloud.fedoraproject.org/results/managerforlustre/buildtools/epel-8-x86_64/01152137-rpmdevtools/rpmdevtools-8.10-7.el8.noarch.rpm
+fi
+if ! yum list installed pgdg-redhat-repo.noarch; then
+    yum install -y https://download.postgresql.org/pub/repos/yum/reporpms/EL-7-x86_64/pgdg-redhat-repo-latest.noarch.rpm
+fi
 yum install -y rpmdevtools git ed epel-release python-setuptools gcc openssl-devel postgresql96-devel
```

Comment:
Might want to remove `rpmdevtools` here.

Reply:
And probably `yum autoremove -y rpmdevtools` below, Because why would we want to reinstall it every time?

Reply:
> Might want to remove rpmdevtools here.

No. The first `yum install` installs the repository, the second installs the package from that repository.

> And probably yum autoremove -y rpmdevtools below, Because why would we want to reinstall it every time?

No idea why it's done this way, the build is pretty brittle so I'm not touching it.

Reply:
I think the build has to be idempotent and re-entrant. As such it will stop being brittle :)

Reply:
> The first yum install installs the repository

Are you sure? It looks like a normal RPM package, and it updates already installed rpmdevtools. The second yum install is indeed adding a repository.

Reply:
> I think the build has to be idempotent and re-entrant.
Yes, but it isn't.
> As such it will stop being brittle :)
No, this is done for some reason unknown to me, so unless we clear that reason I won't touch it. I've been spending hours on debugging obscure build failures last time I tried to dig there.